### ISSUE TITLE
Add error checking for meta[].type

### ISSUE THREAD

**Issue by jwildfire** [RhoInc/web-codebook#305]
Make sure the codebook is giving informative messages if meta[].type doesn't match the actual data. Perhaps just add a mouseover to the missing data summary that gives a listing of what the raw values are for the missing rows.

**Comment by jwildfire** [comment]
Specifying a non-numeric column as numeric definitely creates unexpected output. The screenshot below shows a categorical column ("Sex") being coerced to continuous. 

<img width="1048" alt="screen shot 2018-09-16 at 7 30 15 pm" src="https://user-images.githubusercontent.com/3680095/45604211-0955c500-b9e7-11e8-8b74-c3c4ed0038d8.png">

I *think* we probably just want to treat non-numeric values as missing in this case, but am interested in input from others. @bzkrouse @samussiah What do you think?

**Comment by jwildfire** [comment]
Happy enough with this, but feedback welcome: 

<img width="1071" alt="screen shot 2018-09-16 at 10 38 45 pm" src="https://user-images.githubusercontent.com/3680095/45607465-8857f700-ba01-11e8-8afa-7a20066a092c.png">

**Comment by bzkrouse** [comment]
@jwildfire Would it make sense to ignore the metadata specification in this case?

**Comment by jwildfire** [comment]
I don't think we can really ignore a user specification. I can see an example where the variable has to be numeric (because of whatever specification) and non-numerics should be treated as missing ...

**Comment by jwildfire** [comment]
## Test Notes

Confirm that non-numeric variables are treated as missing when type is set to "continuous". This can be tested on any data set in CAT by toggling a character variable ("RACE", "SEX", etc) to a type of "continuous" on the settings page, and then viewing the missing values as shown in the screenshot above.

**Comment by danedexF5** [comment]
Works as expected
